playbooks/ansible/roles/sit.cephfs/templates/ceph.smb.usersgroups.yml.j2

```diff
@@ -0,0 +1,15 @@
+resource_type: ceph.smb.usersgroups
+users_groups_id: siteacct
+intent: present
+values:
+  users:
+  {%- for account_group in config.accounts +%}
+    {%- for user_name in config.accounts[account_group].users +%}
+      {%- set account = config.accounts[account_group].users[user_name] +%}
+      {%- if account.samba +%}
+    - name: {{ user_name }}
+      password: {{ account.password }}
+      {%- endif +%}
+    {%- endfor +%}
+  {%- endfor +%}
+  groups: []
```

Comment:
The user may have groups. I think we should create them.

Reply:
Ok. I'll consider the group list along with other change requests.

Reply:
Ok

Reply:
Done.